htmd/parameterization/dihedral.py

```diff
@@ -236,32 +197,25 @@ def _objective(self, x, _):
 
         return rmsd
 
-    def _paramsToVector(self, params):
+    def _paramsToVector(self, params, keys):
```

Comment:
What the `key` stands for? Dihedral name, dihedral atom type names?

Reply:
OK! Actually, it is `dihedral_atomtypes`.

Reply:
Renamed